.github/workflows/website.yml

```
@@ -0,0 +1,77 @@
+  push:
+    tags:
+      - 'website*'
+
+permissions:
+  contents: write
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 4

(indentation)